be/src/vec/exec/vaggregation_node.cpp

```diff
@@ -1155,10 +1226,13 @@ Status AggregationNode::_merge_with_serialized_key(Block* block) {
                     auto emplace_result = [&]() {
                         if constexpr (IsPhmapTraits<HashTableType>::value) {
                             if (LIKELY(i + HASH_MAP_PREFETCH_DIST < rows)) {
-                                agg_method.data.prefetch_by_hash(
-                                        hash_values[i + HASH_MAP_PREFETCH_DIST]);
+                                if constexpr (IsParallelPhmapTraits<HashTableType>::value) {
```

Comment:
FYI, maybe we can wrap `IsParallelPhmapTraits` and `IsPreSerializedKeysHashMethodTraits` into one single Traits? Using `IsPreSerializedKeysHashMethodTraits` above but `IsParallelPhmapTraits` here is a bit confused

Reply:
`IsPreSerializedKeysHashMethodTraits` is about `aggregate state`
I will combine `IsParallelPhmapTraits` and `IsPhmapTraits` in one traits.